app/pages/home/sponsorship.jsx

```diff
@@ -7,7 +7,7 @@ import {
     Brick
 } from './section_components.jsx';
 
-import SponsorLogos from '../../components/SponsorLogos.jsx';
+import SponsorLogos from '../../components';
```

Comment:
This breaks the home page currently, change it to 

Import { SponsorLogos } from ‘../../components’;

Or just revert it